cryptofeed/exchange/binance.py

```diff
@@ -177,7 +179,7 @@ def _check_update_id(self, pair: str, msg: dict) -> (bool, bool):
             return
 
         delta = {BID: [], ASK: []}
-        timestamp = msg['E']
+        ts = msg['E']
```

Comment:
I'd leave this as `timestamp` and rename the passed argument to `receipt_timestamp`. That way the naming of the variable is consistent all the way through (since you're passing `timestamp` into `receipt_timestamp` to `book_callback`